backend/app/services/groups/create.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Groups
+  class Create < Core::Service
+    def call(params:, user:)
+      @params = params
+      @user = user
+      group = yield create_group
+      Success(group)
```

Comment:
No need to do `yield` and then wrap result again

The purpose of `yield` is to unwrap the Result if it is Success or bypass all the future logic if the result is Failure.
You could achieve the same results only just by calling `create_group` here